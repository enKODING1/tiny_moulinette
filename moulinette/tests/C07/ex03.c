#include <unistd.h>
#include <stdio.h>
#include <stdlib.h>

char *ft_strjoin(int size, char **strs, char *sep);

int main(void)
{
    char **arr = (char **)malloc(6 * sizeof(char *));

    arr[0] = "Hello";
    arr[1] = "42";
    arr[2] = "Gyeongsan";
    arr[3] = "So";
    arr[4] = "Hot";
    arr[5] = "Place";

    printf("%s\n", ft_strjoin(0, arr, " "));
    printf("%s\n", ft_strjoin(1, arr, " "));
    printf("%s\n", ft_strjoin(2, arr, " "));
    printf("%s\n", ft_strjoin(3, arr, " "));
    printf("%s\n", ft_strjoin(4, arr, " "));
    printf("%s\n", ft_strjoin(5, arr, " "));
    printf("%s\n", ft_strjoin(6, arr, " "));
    printf("%s\n", ft_strjoin(6, arr, ","));
    printf("%s\n", ft_strjoin(6, arr, ", "));
    printf("%s\n", ft_strjoin(6, arr, ", -"));
    return 0;
}
