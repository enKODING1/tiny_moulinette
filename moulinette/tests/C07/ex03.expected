Hello42GyeongsanSoHotPlace

Hello
Hello 42
Hello 42 Gyeongsan
Hello 42 Gyeongsan So
Hello 42 Gyeongsan So Hot
Hello 42 Gyeongsan So Hot Place
Hello,42,Gyeongsan,So,Hot,Place
Hello, 42, Gyeongsan, So, Hot, Place
Hello, -42, -Gyeongsan, -So, -Hot, -Place
